lib/api/1.1/northbound/nodes.js

```diff
@@ -34,28 +35,41 @@ function nodesRouterFactory (
     Logger,
     Promise,
     _,
-    assert
+    assert,
+    Errors
 ) {
     var router = express.Router();
 
     /**
-     * Renders obmSettings property into a node.
+     * Renders obmSettings and sshSettings properties into a node.
      *
      * @param {object} node - node model instance
      * @return {object} Promise that fulfills to the rendered node
      */
     function _renderNodeObmSettings(node) {
         // This function exists to allow backward compatibility of the 1.1 API
-        // with 2.0 API OBM resources.
+        // with 2.0 API OBM and SSH resources.
         assert.object(node, 'node should be a node model instance.');
         assert.ok(!node.obmSettings, 'node should not have obmSettings property');
 
-        node = _.omit(node.toJSON(), 'obms');
-        return Promise.map(waterline.obms.find({node: node.id}), function(obm) {
-           return _.pick(obm.toJSON(), 'service', 'config');
-        })
-        .then(function(obms) {
+        node = _.omit(node.toJSON(), ['obms', 'ibms']);
+
+        var obms = Promise.map(waterline.obms.find({node: node.id}), function(obm) {
+            return _.pick(obm.toJSON(), 'service', 'config');
+        });
+        var sshSettings = waterline.ibms.findByNode(node.id, 'ssh-ibm-service');
```

Comment:
Since this is only for a 1.1 API backwards compatibility edge case, I'm not going to raise an issue. But in any 2.0 controller I would object to hardcoding for specific IBM service types, especially just for doing 'REDACT' which shouldn't be handled at this layer anyways.